packages/fx-core/templates/plugins/resource/aad/auth/bot/csharp/SSO/SsoDialog.cs

```diff
@@ -16,11 +13,18 @@ public class DialogConstants
     internal static readonly string TEAMS_SSO_PROMPT = "TeamsFxSsoPrompt";
 }
 
+public class StoreItem
+{
+    public string eTag;
+}
+
 public class SsoDialog : ComponentDialog
 {
     ILogger<SsoDialog> _logger;
     BotAuthenticationOptions _botAuthOptions;
     Dictionary<string, string> _commandMapping = new Dictionary<string, string>();
+    IStorage _dedupStorage = new MemoryStorage();
```

Comment:
is it possible to use dependency injection to get an IStorage instance? I saw L49 in readme.md already added MemotryStorage, not sure whether we can reuse it via DI.

Reply:
We also need to tell users switch to another storage for production purpose